client/src/components/ContestList/ContestList.tsx

```diff
@@ -8,46 +9,51 @@ import Container from '@material-ui/core/Container';
+  userContests: Contest[];
 }
 
 export default function ContestList({ userContests }: Props): JSX.Element {
-    const classes = useStyles();
+  const classes = useStyles();
 
-    return (
-        <>
-            {/* Write ternary operator checking to see if any data has been recieved, otherwise return default message */}
-            {userContests.length > 0 ? (
-
-                <List className={classes.list}>
-                    {/* map through given data and alter the return inside of the map */}
-                    {userContests.map(contest => (
-                        <>
-                            {console.log(contest)}
-                            <ListItem key={contest.title}>
-                                <Avatar className={classes.avatar} alt="Test Contest Image" variant="square" src={LionTatoo} />
-                                <Grid className={classes.grid} container>
-                                    <ListItemText
-                                        primary={<>
-                                            <Typography className={classes.typography}>{contest.title}</Typography>
-                                        </>}
-                                        secondary={<>
-                                            <Typography>{contest.description}</Typography>
-                                            <Container className={classes.container}>
-                                                <Button className={classes.button}>{contest.prizeAmount}</Button>
-                                            </Container>
-                                        </>} />
-                                </Grid>
-                            </ListItem>
-                        </>
-                    ))}
-                </List>
-
-            ) : (<Typography> None Available </Typography>)}
-
-        </>
-    )
-}
\ No newline at end of file
+  return (
+    <>
+      {/* Write ternary operator checking to see if any data has been recieved, otherwise return default message */}
+      {userContests.length > 0 ? (
+        <List className={classes.list}>
+          {/* map through given data and alter the return inside of the map */}
+          {userContests.map((contest) => (
+            <>
+              <ListItem key={contest.title}>
+                <Avatar className={classes.avatar} alt="Test Contest Image" variant="square" src={LionTatoo} />
+                <Grid className={classes.grid} container>
+                  <ListItemText
+                    primary={
+                      <>
+                        <Typography className={classes.typography}>{contest.title}</Typography>
+                      </>
+                    }
+                    secondary={
+                      <>
+                        <Typography>{contest.description}</Typography>
+                        <Container className={classes.container}>
+                          <Link to={`/contest/${contest._id}`}>
```

Comment:
Button and Link can be combined here :)

Ref: https://stackoverflow.com/questions/38187833/how-to-combine-reactjs-router-link-and-material-ui-components-like-a-button

Reply:
This is fixed.